src/org/thoughtcrime/securesms/ApplicationContext.java

```diff
@@ -127,6 +128,15 @@ public void onCreate() {
 
     rpc.start();
 
+    // [Migration 04/10/23] Chat Background: migrate old custom background
+    String backgroundImagePath = Prefs.getStringPreference(this, "pref_chat_background", "");
+    if (!backgroundImagePath.isEmpty()) {
+      for (int accId : dcAccounts.getAll()) {
+        Prefs.setBackgroundImagePath(this, accId, backgroundImagePath);
+      }
+      Prefs.setStringPreference(this, "pref_chat_background", "");
+    }
```

Comment:
i'd like to have a clear hint that this is temporary and a clear end of the block

```suggestion
    // migrating chat backgrounds, added  04/10/23, can be removed after some versions
    String backgroundImagePath = Prefs.getStringPreference(this, "pref_chat_background", "");
    if (!backgroundImagePath.isEmpty()) {
      for (int accId : dcAccounts.getAll()) {
        Prefs.setBackgroundImagePath(this, accId, backgroundImagePath);
      }
      Prefs.setStringPreference(this, "pref_chat_background", "");
    }
    // /migrating chat backgrounds
```